packages/gatsby-theme-api-docs/src/components/description.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { designSystem, Markdown } from '@commercetools-docs/ui-kit';
+import { markdownFragmentToReact } from '@commercetools-docs/gatsby-theme-docs';
+
+const DescriptionContainer = styled.div`
```

Comment:
I find the specific names much more helpful when inspecting the component tree at runtime, just like we name file names rather specific and not assuming you know the folder path because you often see them standalone with no context.

Reply:
Hmm that's debatable 😄

You name files specifically because it's easier to find them when searching e.g. with the editor.
When you inspect the component tree, you always look for the main component first, then then components you see underneath it are obviously part of the component.

There is also another thing to consider. The generated CSS classes have much longer and repetitive names like this.

Reply:
addressed since it's just a code style preference. Changed a few other existing places too on that occasion for consistency.